tests/mosaic/matmul_test.py

```diff
@@ -31,98 +30,100 @@
 try:
   import hypothesis as hp
   import hypothesis.strategies as hps
+  CAN_USE_HYPOTHESIS = True
```

Comment:
Given what Jake said, I'm wondering why we need this case at all. Can't we just make the import unconditional?

I agree the "skipTest" here is bogus, but maybe we should just make it an unconditional dep instead?

Reply:
> Can't we just make the import unconditional?

We can do this as well. This will make this PR larger.